source/adminguide/ui.rst

```diff
@@ -185,18 +185,19 @@ Users can customize the CloudStack's user interface by means of a configuration
 
 To change the logo, login banner, error page icon, etc. the following details can be edited in config.json:
 
-========== ==================================================
-Property   Description
-========== ==================================================
-apiBase    Changes the suffix for the API endpoint
-docBase    Changes the base URL for the documentation
-appTitle   Changes the title of the portal
-footer     Changes the footer text
-logo       Changes the logo top-left side image
-banner     Changes the login banner image
-error.404  Changes the image of error Page not found
-error.403  Changes the image of error Forbidden
-error.500  Changes the image of error Internal Server Error.
```

Comment:
@utchoang couldn't see this property details here, please check.
http://qa.cloudstack.cloud/docs/WIP-PROOFING/pr/251/adminguide/ui.html#basic-ui-customization

Reply:
@sureshanaparti Do these changes appear to be after you changed the base from master to 4.16?